flask_app/pages/upload.py

```diff
@@ -1,14 +1,36 @@
-from flask import render_template, session, request, redirect, url_for
+import io
+import os
+
+from flask import render_template, session, request, redirect, url_for, send_file
+import humanize
 
 from flask_app.db import get_connection
+
 from flask_app.models.submission import Submission
+import flask_app.models.data_spreadsheet as data_spreadsheet
```

Comment:
Maybe I'm missing something here, but how is it possible to have this working from the `models` folder ? 
The `data_spreadsheet.py` script is under the `legacy` if I am not mistaken..

Reply:
Yeah, but that's because I eventually moved it there, right now this import is different: https://github.com/AndrewRadev/bacterial_growth/blob/e733d69f2955015e10c92b4cc657e60b1c1ddc55/flask_app/pages/upload.py/#L13-L16

Reply:
In this particular PR, it was under models: 
![2024-12-10-163247_1920x1080_scrot](https://github.com/user-attachments/assets/de3b8e2e-853f-4247-9c2e-2d1180082db0)